### ISSUE TITLE
Maybe we should support http2 ?

### ISSUE THREAD

**Issue by ScSofts** [answerdev/answer#161]
Since golang has its official support of http2,  I think we can easily support http2 as a new feature. Sadly, I have never work on  golang, so I cannot do it myself.😭

**Comment by mingcheng** [comment]
Yep, thanks for your suggestion. I found some usage information in which about golang's offical http2 library. And we will have a try: https://posener.github.io/http2/
